lib/active_shipping/carrier.rb

```diff
@@ -92,6 +92,16 @@ def find_tracking_info(tracking_number, options = {})
       raise NotImplementedError, "#find_tracking_info is not supported by #{self.class.name}."
     end
 
+    # Get a list of services available for the a specific route
+    #
+    # @param origin_country [String] The country of origin
```

Comment:
For v2 I think we should have a clear stance on `_name` vs `_code` representations. It's causing headache in our other codes bases. 

@jonathankwok is heading up the vision for v2 and I'd probably say we'd be ok with porting this to the v1.* branch. Kwok, what's your opinion on being explicit about argument representations?

For v2 IMO we should also remove the ambiguity on `ActiveShipping::Location.country`